homeassistant/components/harmony/remote.py

```diff
@@ -57,6 +55,7 @@
         vol.Optional(ATTR_ACTIVITY): cv.string,
         vol.Required(CONF_NAME): cv.string,
         vol.Optional(ATTR_DELAY_SECS, default=DEFAULT_DELAY_SECS): vol.Coerce(float),
+        vol.Optional(ATTR_ACTIVITY_NOTIFY, default=False): cv.boolean,
```

Comment:
Since this is converted to a config flow, we don't update the legacy schema with new options anymore. We should take this out.

Reply:
Removed that 1 line, should the rest for PLATFORM_SCHEMA stay in there or be removed as well?

Reply:
We probably still have people who haven’t upgraded yet so best to leave it in for now